discovery-provider/src/queries/search_queries.py

```diff
@@ -0,0 +1,662 @@
+from enum import Enum
```

Comment:
maybe let's add a clean up item to make this a little more similar to other things in queries, but im good /w this for now for sure

Reply:
yeah, def want to tag out the tag search stuff. will add